modules/virt-adding-templates-to-custom-namespace.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-adding-templates-to-custom-namespace.adoc
+
+[id="virt-adding-templates-to-custom-namespace_{context}"]
+= Adding templates to a custom namespace
+
+The `ssp-operator` deploys Red Hat templates to the `openshift` namespace by default. Templates in the `openshift` namespace are publicly availably to all users. When a custom namespace is created and templates are added to that namespace, virtual machine templates in the `openshift` namespace can be modified or deleted. To add templates to a custom namespace, edit the `hco` custom resource (CR) which contains the `ssp-operator`.
+
+.Procedure
+
+* View the list of Red Hat templates that are available in the `openshift` namespace.
++
+[source,terminal]
+----
+$ oc get templates -n openshift
+----
++
+* Open the `hco` custom resource (CR).
++
+[source,terminal]
+----
+$ oc edit hco -n openshift-cnv kubevirt-hyperconverged
+----
++
+* Add the `commonTemplatesNamespace` attribute and specify the custom namespace. Example:
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec.commonTemplatesNamespace: customnamespace <1>
```

Comment:
Not sure if it works, I suppose to see the same thing in the delete doc, like below:
```
spec:
  commonTemplatesNamespace: customnamespace <1>
````
?

Reply:
@gouyang - please explain. I don't understand your comment. What changes do you want to see here? Requesting your explanation.

Reply:
@lmandavi 
believe means changing:
```
spec.commonTemplatesNamespace:
```
to
```
spec:
  commonTemplatesNamespace:
```
to match other usage in doc

Reply:
Updated the content.